### PR TITLE
Add origin AS filter

### PR DESCRIPTION
**Pull request by digizeph**

### CODE REVIEW

File: lib/bgpstream_elem.h [hunk 1]
```diff
@@ -215,6 +215,12 @@ typedef struct bgpstream_elem {
    */
   bgpstream_as_path_t *as_path;
 
+  /** Prefix originator AS number
+   *
+   * Available only for RIB and Announcement elem types
+   */
+  int32_t origin_asn;
```
Comment (alistairking) [inline lib/bgpstream_elem.h]: what is this for?

Comment (digizeph) [reply lib/bgpstream_elem.h]: ah, not useful anymore. will remove it in next commit.

Comment (digizeph) [reply lib/bgpstream_elem.h]: removed in https://github.com/CAIDA/libbgpstream/pull/46/commits/fb2f8e6eb59d726138f3609de3051aba1092b8c6

**Comment by digizeph** [comment]
@alistairking the merge conflict has been resolved

**Pull request merged**